src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -92,16 +99,30 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $config = $this->getConfig($extension, $container, $input->getOption('resolve-env'));
 
+        $format = $input->getOption('format');
+
+        if ('yaml' === $format && !class_exists(Yaml::class)) {
+            $errorIo->error('Setting the "format" option to "yaml" requires the Symfony Yaml component. Try running "composer install symfony/yaml" or use "--format=json" instead.');
```

Comment:
This is `composer require`

```suggestion
            $errorIo->error('The "yaml" format requires the installation of the Yaml component. Install it with "composer require symfony/yaml", or choose another format by specifying "--format=json".');
```